tests/e2e/specs/BoundaryEventCommonBehaviour.spec.js

```diff
@@ -98,7 +95,7 @@ boundaryEventData.forEach(({ type, nodeType, eventXMLSnippet, eventXMLSnippetWit
         .its('xml')
         .then(removeIndentationAndLinebreaks)
         .then(xml => {
-          const boundaryEventPositionXml = 'bpmndi:BPMNShape id="node_3_di" bpmnElement="node_3"><dc:Bounds x="662" y="481"';
+          const boundaryEventPositionXml = 'bpmndi:BPMNShape id="node_3_di" bpmnElement="node_3"><dc:Bounds x="662" y="471"';
```

Comment:
Not sure why the 10px difference but changing this allowed the test to pass.

Reply:
#741 should hopefully fix this

Reply:
@lisa-fehr awesome. For some reason this change passed on my machine but failed on CircleCI...